internal/clients/filter_instances_test.go

```diff
@@ -28,10 +28,19 @@ func TestMapArchitectures(t *testing.T) {
 	result, err = clients.MapArchitectures(context.Background(), "arm_mac")
 	assert.Nil(t, err)
 	assert.Equal(t, clients.ArchitectureTypeAppleArm64, result)
-	_, err = clients.MapArchitectures(context.Background(), "ios")
+	_, err = clients.MapArchitectures(context.Background(), "ppc64")
 	assert.NotNil(t, err)
 }
 
+func TestAzureMapArchitectures(t *testing.T) {
+	result, err := clients.MapArchitectures(context.Background(), "x64")
+	assert.Nil(t, err)
+	assert.Equal(t, clients.ArchitectureTypeX8664, result)
+	result, err = clients.MapArchitectures(context.Background(), "arm64")
+	assert.Nil(t, err)
+	assert.Equal(t, clients.ArchitectureTypeArm64, result)
+}
```

Comment:
Azure currently have only x86_64 and arm64 is in beta, at least I don't see any instance types reported with this architecture.